src/WebJobs.Script.WebHost/Models/EncryptedHostAssignmentContext.cs

```diff
@@ -14,20 +13,5 @@ public class EncryptedHostAssignmentContext
 
         [JsonProperty("isWarmup")]
         public bool IsWarmup { get; set; }
-
-        public static EncryptedHostAssignmentContext Create(HostAssignmentContext context, string key)
-        {
-            string json = JsonConvert.SerializeObject(context);
-            var encryptionKey = Convert.FromBase64String(key);
-            string encrypted = SimpleWebTokenHelper.Encrypt(json, encryptionKey);
-
-            return new EncryptedHostAssignmentContext { EncryptedContext = encrypted };
-        }
-
-        public HostAssignmentContext Decrypt(string key)
-        {
-            var decrypted = SimpleWebTokenHelper.Decrypt(key.ToKeyBytes(), EncryptedContext);
```

Comment:
Centralized this decryption logic into the StartupContextProvider. That way the same decryption logic is used for both Windows and Linux Consumption.